temporian/core/event_set_ops.py

```diff
@@ -2387,6 +2387,146 @@ def tick(
 
         return tick(self, interval=interval, align=align)
 
+    def tick_calendar(
+        self: EventSetOrNode,
+        second: Union[int, Literal["*"], None] = None,
```

Comment:
Use "Optional" instead of Union[..., None].

Reply:
Please fix ?

Reply:
This is already fixed, for some reason github shows the old version. The new one is here:

https://github.com/google/temporian/pull/255/files#diff-c533efec938d9d3b45600f0d233ca722232b26f02e2e8196f680c49c752733e7R2390